scripts/errcheck_excludes.txt

```diff
@@ -1,2 +1,2 @@
 // Never check for logger errors.
-(github.com/go-kit/kit/log.Logger).Log
+(github.com/go-kit/log.Logger).Log
```

Comment:
Some help here will be required. Thank you.

Reply:
This is fine, it's a bit of a weird issue. The errors returned by `.Log` in this library are not useful, as it's all handled at compile time. So it the linter is miss-reporting things.